### ISSUE TITLE
`Error reading file: Is a directory` when requiring

### ISSUE THREAD

**Issue by hedgehog1029** [veelenga/lua.cr#24]
Hey, I'm a bit new to Crystal so this may be me being dumb, but I'm trying to use lua.cr and I get this error when requiring it:
```
Error in src/ultra.cr:4: while requiring "./discord/plugins.cr"

require "./discord/plugins.cr"
^

in src/discord/plugins.cr:1: while requiring "lua": Error reading file: Is a directory

require "lua"
^
```
I'm not sure where the error is coming from?

**Comment by veelenga** [comment]
That's my mistake with shard names. Fixed by c3ea93dcd41cee24030b1afb27e7e6fcd6226c93.

You need to do now:

1. Remove lua.cr from your `lib/` folder (in your discord repository).
2. Remove lua.cr entry from `shard.lock` if any.
2. Change a lua dependency in `shard.yml` to:

```yaml
dependencies:
  lua:
    github: veelenga/lua.cr
```
4. Do `crystal deps` and try again.

Thanks for trying it. Do not hesitate to ask in case of any issues ;)

**Comment by hedgehog1029** [comment]
Can confirm this was fixed by following the above steps; thank you!
